apps/examples/testcase/le_tc/kernel/tc_termios.c

```diff
@@ -45,6 +45,18 @@ static void tc_termios_tcsetattr_tcgetattr(void)
 	}
 
 	sleep(1);
+	ret_chk = isatty(fileno(stdin));
+	TC_ASSERT_EQ("isatty", ret_chk, 0);
```

Comment:
I followed the test guide and yes, isatty() TC failed.
It's bcz, isatty() returns 1 for SUCCESS, 0 for FAILURE. So I was checking equal to zero condition.
It's fixed.